app/Services/Campaign/DefaultImageService.php

```diff
@@ -88,17 +88,18 @@ public function destroy(): bool
      */
     public function destroyAll(): void
     {
-        $images = $campaign->default_images ?? [];
+        $images = $this->campaign->default_images ?? [];
 
-        foreach ($images as $img) {
+        foreach ($images as $key => $img) {
```

Comment:
If we have a placeholder image for each module + for 10 custom modules, this is 35 db requests to get the image and 35 to delete it, when it could be just a single one for everything.